dffml/model/model.py

```diff
@@ -89,10 +89,10 @@ def __init__(self, config):
         if isinstance(location, pathlib.Path):
             # to treat "~" as the the home location rather than a literal
             location = location.expanduser().resolve()
-            # TODO Change all model configs to make them support mutable
-            # location config properties
-            with self.config.no_enforce_immutable():
-                self.config.location = location
+        # TODO Change all model configs to make them support mutable
+        # location config properties
+        with self.config.no_enforce_immutable():
+            self.config.location = location
```

Comment:
Why has this been moved in 4 spaces?

Reply:
I moved it out 4 spaces, I am not sure why it was within the ` if isinstance(location, pathlib.Path):` body, but it was creating problems in some test cases where this block was not executed, leaving the `self.config.location` attribute  a string whereas we expect it to be a `pathlib.Path`